test/end-to-end/processor/_normalizer.xsl

```diff
@@ -81,7 +81,7 @@
 				in:		<a href="file:/path/to/tested.xsl">/path/to/tested.xsl</a>
 				out:	<a href="tested.xsl">tested.xsl</a>
 	-->
-	<xsl:template as="element(a)" match="/html/body/p[1]/a" mode="local:normalize">
+	<xsl:template as="element(a)" match="/html/body/p[1 or 2]/a" mode="local:normalize">
```

Comment:
Meant to be `[position() = (1, 2)]`?

Reply:
Both predicates work. I've changed this to the more explicit version as suggested.